autogpt_platform/frontend/src/components/monitor/scheduleTable.tsx

```diff
@@ -198,6 +240,7 @@ export const SchedulesTable = ({
                     {agents.find((a) => a.id === schedule.graph_id)?.name ||
```

Comment:
Should we just append the version here? e.g: {graph_id}#V{graph_version} instead of a new column ?